Git-Credential-Manager.sln

```diff
@@ -199,13 +199,9 @@ Global
 		{8DBBAB0A-970D-4BE3-958C-8CDC92F76549}.WindowsRelease|Any CPU.ActiveCfg = Release|Any CPU
 		{8DBBAB0A-970D-4BE3-958C-8CDC92F76549}.WindowsRelease|Any CPU.Build.0 = Release|Any CPU
 		{4C2DBC8A-B3F2-4C64-870A-BA79DA4BD403}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{4C2DBC8A-B3F2-4C64-870A-BA79DA4BD403}.Debug|Any CPU.Build.0 = Debug|Any CPU
```

Comment:
Perhaps this deletion is why I didn't get the Windows installer on the default build. The `Release|Any CPU` line is similar.

Reply:
See my answer to your build question.